files/common/usr/bin/get-hotfix-version

```diff
@@ -0,0 +1,37 @@
+#!/bin/bash
+#
+# Copyright 2020 Delphix
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+set -o errexit
+set -o pipefail
+
+function usage() {
+	echo "Usage: $(basename "$0")"
+	echo
+	echo "Display the hotfix version information for the appliance."
+	exit 2
+}
+
+if [[ $# -gt 0 ]]; then
+	echo "No arguments are supported."
+	usage
+fi
+
+output=$(zfs get -Hpo value "com.delphix:hotfix-version" "$(dirname "$(zfs list -Hpo name /)")")
+if [[ "$output" != "-" ]]; then
+	echo -n "$output"
+fi
+
```

Comment:
trailing line

Reply:
We probably need these script checkers are part of this repo so the developer can run them rather than relying on reviewers and these checkers entirely. It dirty ups the review to have these revisions rather than doing it on the dev's machine.

Reply:
I understand the concern, and for these simple checks that may be a reasonable possibility, but for more complicated checks that may not always be achievable (e.g. in other repositories, we do more complicated checks, and duplicating that test environment outside of github actions would not be possible); that is, unless the actions can be run locally.